src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        AbstractFigure[] figures = new AbstractFigure[6];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+            if (figures[i] != null) {
+                System.out.println(figures[i].draw());
+            }
+        }
+    }
+}
+
+
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```